src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
```

Comment:
It seems like there's an empty comment here. If it's not being used, it should be removed to keep the code clean.